github_tutorial.md

```diff
@@ -22,22 +26,22 @@ Er kunnen tegelijkertijd veel verschillende branches actief zijn, zodat verschil
 Voor een standaard die releasematig wordt beheerd en gepubliceerd, zal er voor elke versie (incl. patchversies) een branch worden gemaakt, waarin de wijzigingen voor die versie worden toegepast.
 Voor iemand die een wijziging of toevoeging wil doorvoeren, is het verstandig voor elke wijziging/toevoeging een eigen branch te maken, zodat ook pull requests elk slechts één wijziging/toevoeging bevat.
 
-**master**
-Er is één speciale branch, "master". Dit is de basis, de actueel geldende versie van de repository. Wijzigingen voeg je niet direct toe op de "master"-branch, maar op een aparte branch voor deze wijziging.
-Pas wanneer de wijziging klaar is en kan worden gepubliceerd, voeg je de branch met de wijziging samen met de master-branch (dit heet `merge`), zodat de wijzigingen worden doorgevoerd op de master.
+Enkele branches zijn bijzonder, te weten:
+
+* **master** Dit is de basis, de actueel geldende versie van de repository.
+* **develop** Hier vind actieve ontwikkeling plaats aan de volgende, maar nog niet officiele, versie.
```

Comment:
Wordt er nu een develop branch gebruikt? Die vindt ik niet terug in bijvoorbeeld Gemma-zaken (en trouwens ook niet in bevragingen-ingeschreven-personen)

Reply:
Oeps, ik had deze gemist.

Klopt. Wellicht goed om het te laten zoals je het had. Bij code ontwikkelingen onderscheiden we `develop` en `master`. Typisch omdat we `develop` op een (interne) testomgeving hebben, en `master` op een echte (in dit geval externe test-) omgeving.

Wellicht is het beter om `master` voor ontwikkeling te houden en dan speciale branches voor releases aan te houden: `stable/1.0.x`, `stable/1.1.x`. Patches (de `x`) worden dan op deze branches toegevoegd, terwijl er ook aan andere versies nog gewerkt kan worden.